src/main/java/apoc/util/Util.java

```diff
@@ -2,10 +2,11 @@
 
 import apoc.ApocConfiguration;
 import apoc.Pools;
-import apoc.export.util.*; // todo
+import apoc.export.util.CountingInputStream;
```

Comment:
My fault. I always ask my guys to do "reorganize import" before committing, in order to remove warnings for useless imports.